CHANGES.md

```diff
@@ -14,6 +14,7 @@
 
 - Only logged in users can execute functions (for guests, use anonymous login)
 - Only the user who has triggered the execution get access to the relevant execution logs
+- Function execution env `APPWRITE_FUNCTION_EVENT_PAYLOAD` renamed to `APPWRITE_FUNCTION_ENV_DATA`
```

Comment:
to `APPWRITE_FUNCTION_EVENT_DATA`

Reply:
I would also list this change under breaking changes.

Reply:
This line currently under `0.8 Breaking Changes`. Is there another place to list breaking changes?
https://github.com/appwrite/appwrite/blob/456793ebd8dc92e67b5643d2e95775a5da88f6e8/CHANGES.md#breaking-changes

Reply:
Sorry my bad, had the wrong context view on the file changes